test/unit_tests/modules/util/validators-spec.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+var chai = require( 'chai' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+var validators = require(
+  '../../../../cfgov/unprocessed/js/modules/util/validators.js'
+);
+var ERROR_MESSAGES = require(
+  '../../../../cfgov/unprocessed/js/config/error-messages-config'
```

Comment:
You could use a `var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';` constant at the top of the file, as is done in some other [specs](https://github.com/cfpb/cfgov-refresh/blob/flapjack/test/unit_tests/modules/util/assign-spec.js#L3).